WalletWasabi.Backend/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  postgres:
+    image: postgres:13.1
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes:
+      - "postgres_datadir:/var/lib/postgresql/data"
+    ports:
+      - "65466:5432"
+    expose:
+      - "5432"
+volumes:
+  postgres_datadir:
```

Comment:
weird to me that there is no newline but I guess it's just creating a volume and that's it?